modules/oe_starter_content_publication/config/install/field.storage.node.oe_sc_publication_document.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
```

Comment:
so, I looked into it also, the other oe_document storages did not match requirements, had either a different cardinality or a different target_type